gc.go

```diff
@@ -33,7 +33,87 @@ type Config struct {
 	WatchDir              string `json:"watch_dir"`
 }
 
+func setupConfig() *Config {
+	var config Config
+
+	configFiles := []string{"${XDG_CONFIG_HOME}/gc/config",
+		"${HOME)/.config/gcrc",
+		"${HOME}/.gcrc"}
+
+	var path string
+	for _, path = range configFiles {
+		path = os.ExpandEnv(path)
+		_, err := os.Stat(path)
+		if err == nil {
+			configFile, err := os.Open(path)
+
+			if err != nil {
+				panic(err)
+			}
+
+			err = json.NewDecoder(configFile).Decode(&config)
+
+			if err != nil {
+				panic(err)
+			}
+
+			break
+		} else {
+			path = ""
```

Comment:
This is used to bail out early if gc is run with --no-interactive and no config file is present, but I agree this is weird. I'll remove that.